src/test/java/leetcode/fibonacci/FibonacciTest.java

```diff
@@ -0,0 +1,34 @@
+package leetcode.fibonacci;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+
+
+class FibonacciTest {
+    @Test
+    public void shouldTestFibonacci() {
+        Fibonacci fibonacci = new Fibonacci();
+        int input1 = 2;
```

Comment:
1. Każdy taki input - output ot oddzielny test. Lub użyj @ParametrizedTest
2. Co daje Ci sprawdzenie 6 przypadków? Dla fibonnacciego potrzbeuejsz sprawdzic 3 przypadki: input  =0, input = 1, input => jakaś liczba większa od 1 najlepiej też więszka niż 2. Testy to też kd, który trzeba utrzymywać, nie warto robić bezsensownych testów.